core/src/main/java/org/apache/accumulo/fate/zookeeper/ZooLock.java

```diff
@@ -38,9 +39,27 @@
 import org.slf4j.LoggerFactory;
 
 public class ZooLock implements Watcher {
-  private static final Logger log = LoggerFactory.getLogger(ZooLock.class);
+  private static final Logger LOG = LoggerFactory.getLogger(ZooLock.class);
 
-  public static final String LOCK_PREFIX = "zlock-";
+  private static final String ZLOCK_PREFIX = "zlock#";
+  private static final String ZLOCK_UUID = UUID.randomUUID().toString();
```

Comment:
Since this is static seems like two ZooLocks created in the same process would have the same UUID. Probably does not matter because all code only creates a single zoolock per process, I think.  But it still seems iffy.

Reply:
That was a question that I had, should these be static? I believe we currently only have one ZooLock per VM. I left it static to generate a discussion about it.  I think it's fine as is and easy enough to change in the future.

Reply:
I looked in the code a bit, the Master and GC code created ZooLock objs in a loop.  Could possibly require passing a UUID into the ZooLock constructor.  Maybe this is useful for test and for making things crystal clear in the code.  Like the GC code could created the UUID outside of the loop and it would make what is going on more apparent.

Reply:
Great suggestion, this also removes the need for the conversation about whether or not the UUID should be static. I updated the code and implemented this in the latest commit.

Reply:
I had forgot that UUID.fromString() existed until I saw it used in the update, that made it easy to use the UUID type  in the test.